packages/vats/src/localchain.js

```diff
@@ -63,6 +63,18 @@ const prepareLocalChainAccount = zone =>
         const purse = E(bank).getPurse(allegedBrand);
         return E(purse).deposit(payment);
       },
+      /**
+       * Withdraw a payment from the account's bank purse that matches the
+       * alleged brand.
```

Comment:
```suggestion
       * Withdraw a payment from the account's bank purse of the amount's brand.
```

The caller isn't alleging something in such a way that they might be lying.